### PR TITLE
Add changes for edge-21.11.2

### PR DESCRIPTION
**Pull request by kleimkuhler**
Signed-off-by: Kevin Leimkuhler <kevin@kleimkuhler.com>

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+* Deprecated support for topology keys and added support for topology aware
+  hints
+* Added `logFormat` and `logLevel` configuration values for the `proxy-init`
+  container (thanks @gusfcarvalho!)
+* Added services to the web dashboard (thanks @krzysztofdrys!)
+* Updated commands in the web dashboard to use the `viz` subcommand when
+  necessary (thanks @mikutas!)
+* Added support for building the proxy using a `boringssl` backend instead of
+  the default `rustls` backend
+* Fixed identity overrides for endpoint slices
```
Comment (adleong) [inline CHANGES.md:30]: we can omit this, I think.  this was basically only used by mutlicluster+endpoint slices but that feature still doesn't work end to end yet.

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+  the control plane namespace, leading to issues with installations that use a
+  non-default namespace name
+* Added support for ephemeral storage requests and limits configured either
+  through the CLI or annotations (thanks @michaellzc!)
+* Deprecated support for topology keys and added support for topology aware
+  hints
+* Added `logFormat` and `logLevel` configuration values for the `proxy-init`
+  container (thanks @gusfcarvalho!)
+* Added services to the web dashboard (thanks @krzysztofdrys!)
+* Updated commands in the web dashboard to use the `viz` subcommand when
```
Comment (adleong) [inline CHANGES.md:26]: maybe specify "example commands"

File: CHANGES.md [hunk 3]
```diff
@@ -1,5 +1,36 @@
 # Changes
 
+## edge-21.11.2
+
+This edge release introduces a new Services workload in the web dashboard that
```
Comment (adleong) [inline CHANGES.md:5]: services aren't really workloads.  maybe "Service page"?

**Review by adleong — APPROVED** [review]

**Review by adleong — APPROVED** [review]

File: CHANGES.md [hunk 4]
```diff
@@ -1,5 +1,35 @@
...
+  through the CLI or annotations (thanks @michaellzc!)
+* Deprecated support for topology keys and added support for topology aware
+  hints
+* Added `logFormat` and `logLevel` configuration values for the `proxy-init`
+  container (thanks @gusfcarvalho!)
+* Added services to the web dashboard (thanks @krzysztofdrys!)
+* Updated example commands in the web dashboard to use the `viz` subcommand when
+  necessary (thanks @mikutas!)
+* Added support for building the proxy using a `boringssl` backend instead of
+  the default `rustls` backend
```
Comment (olix0r) [inline CHANGES.md:29]:
```suggestion
```

not user facing

**Review by olix0r — APPROVED** [review]

**Review by hawkw — APPROVED** [review]

**Pull request merged**